tuna/export_db.py

```diff
@@ -46,34 +46,44 @@
 # Setup logging
 LOGGER = setup_logger('export_db')
 
+_, ID_SOLVER_MAP = get_id_solvers()
+
 
 def parse_args():
   """Function to parse arguments"""
   parser = setup_arg_parser('Convert MYSQL find_db to text find_dbs' \
     'architecture', [TunaArgs.ARCH, TunaArgs.NUM_CU, TunaArgs.VERSION])
-  parser.add_argument('-c',
-                      '--opencl',
-                      dest='opencl',
-                      action='store_true',
-                      help='Use OpenCL extension',
-                      default=False)
-  parser.add_argument(
+
+  group_ver = parser.add_mutually_exclusive_group(required=True)
+  group_ver.add_argument(
       '--session_id',
-      action='store',
-      type=int,
       dest='session_id',
+      type=int,
       help=
       'Session ID to be used as tuning tracker. Allows to correlate DB results to tuning sessions'
   )
+  group_ver.add_argument(
+      '--golden_v',
+      dest='golden_v',
+      type=int,
```

Comment:
should we make this required=True?

Reply:
session_id and golden_v are in a mutually exclusive group. Exactly 1 is required.